From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alex Ciarlillo <alex.ciarlillo@gmail.com>
Date: Thu, 23 Mar 2023 11:52:03 -0400
Subject: extend num supported gamepads

Guilded Path: Extended the maximum number of supported gamepad devices

diff --git a/device/gamepad/public/cpp/gamepads.h b/device/gamepad/public/cpp/gamepads.h
index 62b2ecada5e7dfd6bfada99ceb7d9e61b9af3b2e..80353a4ac020b5379e93ec1963c5ff1fe527bbca 100644
--- a/device/gamepad/public/cpp/gamepads.h
+++ b/device/gamepad/public/cpp/gamepads.h
@@ -16,7 +16,7 @@ namespace device {
 // browser.
 class COMPONENT_EXPORT(GAMEPAD_PUBLIC) Gamepads {
  public:
-  static constexpr size_t kItemsLengthCap = 4;
+  static constexpr size_t kItemsLengthCap = 8;
 
   // Gamepad data for N separate gamepad devices.
   Gamepad items[kItemsLengthCap];
